public/locales/send.en-US.ftl

```diff
@@ -4,10 +4,11 @@ uploadPageHeader = Private, Encrypted File Sharing
 uploadPageExplainer = Send files through a safe, private, and encrypted link that automatically expires to ensure your stuff does not remain online forever.
 
 uploadPageLearnMore = Learn more
-uploadPageDropMessage = Drop your files here to start uploading
+uploadPageDropMessage = Drop your file here to start uploading
```

Comment:
Once we have other translations we'll need to change the key when we need to make changes to the string like this. For example if we were to change it back to "files" in v2 we'd rename the key to something like`uploadPageDropMultipleMessage` or `uploadPageDropMessage2`.